Disaster Recovery/SafeHaven 5 DCCF to DCCF/master-dccf-dccf.md

```diff
@@ -0,0 +1,63 @@
+{{{
+  "title": "Master Article - DCCF(Production) to DCC-F(DR) - SafeHaven Setup",
```

Comment:
DCCF/DCC-F is being renamed to CenturyLink Private Cloud on VMware Cloud Foundation on 5-17
Adjust accordingly.